README.md

```diff
@@ -12,3 +12,13 @@ CockroachDB does not support `ENUM` types, so using `Sequelize.ENUM`s with
 CockroachDB will be backed by `TEXT` fields. This means that there will be no
 database-level assurance that the fields are valid (though Sequelize itself
 will validate values if `typeValidation` is enabled).
+
+## Setup and run tests
+
+First make sure you have CockroachDB installed. You can run `cockroach version` to see if is installed or you can [download here](https://www.cockroachlabs.com/docs/stable/install-cockroachdb.html)
+
+Run `cockroach start-single-node --insecure --logtostderr` to start the database
```

Comment:
I ran into this error `ERROR: cockroach server exited with error: unable to lookup hostname` if I didn't provide a hostname, maybe note this with and say `--host localhost` will fix this if you run into it.

https://github.com/cockroachdb/cockroach/issues/23461

Reply:
ok, I will add this note